### DISCUSSION TITLE
2FA 2 TOTP not working ?

### DISCUSSION THREAD

**Discussion by oserin** [community/community#147499] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Copilot

### Body

I am using microsoft authenticator on one mobile and authy on another one, I can sign in with microsoft one but the authy is not working ...
Any clue on that ?

**Comment by saroroce** [comment]
It's a bug they will fix it just wait sometime

**Reply by oserin** [reply]
Hi thx for the answer, I could not find anything related in my searches - it would be awesome if you have a ticket reference !

**Reply by saroroce** [reply]
> Hi thx for the answer, I could not find anything related in my searches - it would be awesome if you have a ticket reference !

Sorry, i don't have it

**Comment by shinybrightstar** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @oserin ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by oserin** [reply]
Hey shiny, AI helped people get civilized ;) Thx for handling this properly !

**Reply by shinybrightstar** [reply]
Hello @satri35 

This Accessibility category is for users to discuss and provide feedback on the accessibility of GitHub products, which our teams will review. You can find more info in this [doc](https://docs.github.com/en/get-started/accessibility)

**Comment by MasterInQuestion** [comment]
Refer:
    https://github.com/orgs/community/discussions/140103#discussioncomment-11125293
    (Time matters for TOTP)

**Reply by oserin** [reply]
Well I dont think that I have the same issue ? I understand fro that post that time matters and some app handle better than others ? I also tried using 2FA (though in github Authy is also suggested) and this still does not work on that mobile ? I am thinking that this could be mobile dependent ? Also I am wondering should I theoretically have the same code displayed on both mobile/apps because this is not the case.

**Reply by MasterInQuestion** [reply]
The implementation of TOTP is simple straightforward:
    https://github.com/orgs/community/discussions/63813#discussioncomment-11120840
    .
    If the implementation properly handles time zone: (most shall do?)
    The failure source would be the time itself.
    (sync the time between the devices)
